scripts/.babelrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Don't use a `.babelrc` file please.
We move all configuration into `package.json`